gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
Hm do you mean I don't need the java part with srcDir = []? Or that I dont need to add the java source folder to scala srcDirs? Because I tried it without adding "src/main/java" and was getting compilation errors.